coursebuilder/views/unit.html

```diff
@@ -126,7 +126,7 @@
                 {% if next_button_url %}
                   <a href="{{ next_button_url }}" class="btn right">{{ gettext('Next') }}</a>
                 {% else %}
-                  {% if unit.title in course_outline[((course_outline|length) - 1)].title %}
+                  {% if unit_id == course_outline[((course_outline|length) - 1)].id %}
```

Comment:
@Tawakalt Are we checking if the unit.id == ( course_outline.length - 1 ) ? Is there a case where we have a private unit whose lessons do not show on the the outline and how would that play out here?

Reply:
@chegejames I didn't consider that, let me look into it.

Reply:
@chegejames We are checking if the current unit.title == the title of the last element in course_outline. If a private unit is the last unit, then the unit before it is considered as the last for people without access.